driver-core/src/main/com/mongodb/operation/AggregateToCollectionOperation.java

```diff
@@ -120,8 +120,6 @@ public AggregateToCollectionOperation(final MongoNamespace namespace, final List
         this.aggregationLevel = notNull("aggregationLevel", aggregationLevel);
 
         isTrueArgument("pipeline is not empty", !pipeline.isEmpty());
-        isTrueArgument("last stage of pipeline contains an output collection",
```

Comment:
I don't think it's worth having this check here, so rather than expanding it to cover $merge I've removed it.

Reply:
Ack